packages/dev/core/src/Engines/WebGL/webGLRenderTargetWrapper.ts

```diff
@@ -142,7 +176,8 @@ export class WebGLRenderTargetWrapper extends RenderTargetWrapper {
      * @param lodLevel defines the lod level to bind to the frame buffer
      */
     private _bindTextureRenderTarget(texture: InternalTexture, attachmentIndex: number = 0, faceIndexOrLayer?: number, lodLevel: number = 0) {
-        if (!texture._hardwareTexture) {
+        const hardwareTexture = texture._hardwareTexture as WebGLHardwareTexture;

```

Comment:
Not sure why the change is necessary ?

Reply:
Just because `texture._hardwareTexture` is used several times (5) in the function.